### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by acperez**

### CODE REVIEW

File: examples/simple_notification/webapp/js/push.js [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+navigator.mozPush.requestURL("sds","dssd");
+
+    this.clearButton = document.getElementById('buttonClear');
+    this.clearButton.addEventListener('click', function(){
+      debug("HOLALALALAAL");
+      navigator.mozPush.getCurrentURL();
+    });
+
+
+ navigator.mozApps.getSelf().onsuccess = function getSelfCB(evt) {
```
Comment (frsela) [inline examples/simple_notification/webapp/js/push.js]: NIT: Fix ident

File: examples/simple_notification/webapp/js/push.js [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+var Push = {
+
+  init: function() {
+
+navigator.mozPush.requestURL("sds","dssd");
+
+    this.clearButton = document.getElementById('buttonClear');
+    this.clearButton.addEventListener('click', function(){
+      debug("HOLALALALAAL");
```
Comment (frsela) [inline examples/simple_notification/webapp/js/push.js]: Better debug traces :D

File: examples/simple_notification/webapp/js/push.js [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+    this.database;
+
+    this.registerAppButton = document.getElementById('buttonRegisterApp');
+    this.getURLButton = document.getElementById('buttonGetURL');
+    this.clearButton = document.getElementById('buttonClear');
+    this.logArea = document.getElementById('logarea');
+
+/*    this.registerAppButton.addEventListener('click', this.registerApp.bind(this));
+    this.getURLButton.addEventListener('click', this.getURL.bind(this));
+    this.clearButton.addEventListener('click', this.onclear.bind(this));*/
```
Comment (frsela) [inline examples/simple_notification/webapp/js/push.js]: Why commented?

File: examples/simple_notification/webapp/js/push.js [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+};
+
+window.addEventListener('load', function pushOnLoad(evt) {
+  window.removeEventListener('load', pushOnLoad);
+  Push.init();
+});
+
+window.addEventListener('unload', function pushOnUnLoad(evt) {
+  window.removeEventListener('unload', pushOnUnLoad);
+debug("bye byebyebeybeyebey");
```
Comment (frsela) [inline examples/simple_notification/webapp/js/push.js]: same as before ;)

File: examples/simple_notification/webapp/manifest.webapp [hunk 5]
```diff
@@ -0,0 +1,38 @@
+{
+  "name": "Push",
+  "description": "Push Notifications",
+  "type": "certified",
+  "launch_path": "/index.html",
+  "developer": {
+    "name": "Fernando Rodriguez Sela, Guillermo López Leal",
```
Comment (frsela) [inline examples/simple_notification/webapp/manifest.webapp]: And you ;)

File: examples/simple_notification/webapp_server/src/es/tid/push/Register.java [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+					clientListener.onNewClientRegistered(value);
+				else{
+					clientListener = (RegistrationListener)getServletContext().getAttribute("clientListener");
+					if(clientListener != null)
+						clientListener.onNewClientRegistered(value);
+				}
+			} catch (MalformedURLException e) {
+				response.sendError(400, "Push_url malformed");
+			}
+	    }
```
Comment (frsela) [inline examples/simple_notification/webapp_server/src/es/tid/push/Register.java]: indent

File: examples/simple_notification/webapp/index.html [hunk 7]
```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html manifest="manifest.appcache">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="pragma" content="no-cache">
+    <title>TestApp</title>
+
+    <link rel="stylesheet" href="style/push.css" type="text/css" />
+<script type="text/javascript" src="shared/js/desktop.js"></script>
```
Comment (willyaranda) [inline examples/simple_notification/webapp/index.html]: indent

File: examples/simple_notification/webapp/js/push.js [hunk 8]
```diff
@@ -0,0 +1,242 @@
...
+    return;
+  //console.log('[DEBUG] PushApp: ' + msg);
+  dump('[DEBUG] PushApp: ' + msg + '\n');
+}
+
+var Push = {
+
+  init: function() {
+
+navigator.mozPush.requestURL("sds","dssd");
```
Comment (willyaranda) [inline examples/simple_notification/webapp/js/push.js]: callbacks for this DOMRequest?

File: examples/simple_notification/webapp/js/push.js [hunk 9]
```diff
@@ -0,0 +1,242 @@
...
+    this.initStore(function (success, error){
+      if (success) {
+        this.watoken = success.watoken;
+        this.pbk = success.pbk;
+      }
+
+      debug("watoken: " + this.watoken);
+      debug("pbk: " + this.pbk);
+
+      if (this.watoken == null || this.pbk == null) {
```
Comment (willyaranda) [inline examples/simple_notification/webapp/js/push.js]: should be `(!this.watoken || !this.pbk)`.

**Pull request merged**